### DISCUSSION TITLE
Can KasmVNC do multiple monitors?

### DISCUSSION THREAD

**Discussion by dstromberg** [kasmtech/KasmVNC#157] [Q&A]
Hi.

I'm very interested in KasmVNC.

About the only thing stopping me from using it a LOT, is I can't figure out how to get it to do multihead.  For work I'm using a Mac laptop with 2 external displays, and I about can't use Kasm if it won't use all 3 well.

But KasmVNC is about the only thing I've seen for remote display that does microphone passthrough, making it a strong candidate.

Can KasmVNC do multihead?  I'd be overjoyed to use a different browser or something if that'd help.

Thanks!

**Comment by real-limitless** [comment]
Vmware Horizon implemented this successfully. 

A function in the OS side would need to be done on the OS to attach a extra display.
Then on the web a function to attach another window to be the same in the same session and stream only seperate displays.

**Comment by mmcclaskey** [comment]
We are actively developing this, it will most likely be done by the next release and we will have a developer preview as soon as it is merged into the master branch. I will follow up on this thread once the developer preview is ready.

**Reply by real-limitless** [reply]
That is perfect @mmcclaskey++!
I would love to see it in developer preview.

**Comment by sbpro86** [comment]
What's new about this important feature ?

**Comment by mmcclaskey** [comment]
Multi-monitor support was added a versions back, apologies for not updating this discussion.

**Comment by sbpro86** [comment]
It is a great feature, thank you.

Does it also work for Windows ?

@mmcclaskey

**Reply by mmcclaskey** [reply]
KasmVNC does not currently support Windows.

**Reply by mmcclaskey** [reply]
I should say, KasmVNC doe snot currently support Windows as the server. Yes, multi-monitor works on Windows clients.
